### ISSUE TITLE
Big Picture Mode Not Working on AMD

### ISSUE THREAD

**Issue by OSCrustacean** [ValveSoftware/steam-for-linux#9487]
#### Your system information

* Steam client version (build number or date):   1682708537 (April 28, 2023)
* Distribution (e.g. Ubuntu): Fedora 38
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Big picture mode flashes black screen repeatedly but does not open. Should open as expected. I migrated to the RPM version because I thought this issue might have something to do with the Flatpak (originally experienced issue with client from Flathub), and I also reproduced it on Opensuse Tumbleweed, so I am comfortable saying it is with the underlying client. I am using an AMD Ryzen 7 7700X CPU and an AMD Radeon RX 5700XT with Mesa driver version 23.0.3, so this isn't the issue with NVIDIA.

#### Steps for reproducing this issue:

1. Install steam from Fedora repos
2. Try to run big picture mode every time except the first time opening the client (e.g. exit client and reopen after install)
3. Big picture mode will not launch
[bigPictureCrash.webm](https://github.com/ValveSoftware/steam-for-linux/assets/30218744/370c0a28-c0c2-44dc-9707-d3b80b92b328)

**Comment by kisak-valve** [comment]
Hello @OSCrustacean, if you completely close Steam, then run `steam` from a terminal, does that have an effect? Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by jntesteves** [comment]
Hello, @kisak-valve, I've also had the same issue on Fedora GNOME Wayland + AMD GPU for a few months and as you describe launching Steam from a terminal solves the issue.

I just want to add that I've just switched to the new Steam Client Beta, and now this issue happens not only for BPM, but also when launching the desktop client. Now, with the new framework, Steam can't create any window if not launched from a terminal.

[Steam Client Beta System Information 2023-05-14](https://gist.github.com/jntesteves/9c7bf4bd793cd86db8c3314cd0714ab6)

**Comment by OSCrustacean** [comment]
Yes Steam BPM works when launched from the terminal, and also I experienced the same issue with the Beta as mentioned above. I included a link to my gist above.
